vitrina/uapi/templates/requests/request_detail.html

```diff
@@ -0,0 +1,117 @@
+            <div class="column is-7">
+                <a href="{% url 'agent-detail' organization.pk request_history.agent.pk %}" class="has-text-link" target="_blank">
+                {{ request_history.agent.title }}
+                    <span class="icon is-small"><i class="fas fa-external-link-alt"></i></span>
+                </a>
+            </div>
+        </div>
+
+        <div class="columns is-mobile mb-1">
+            <div class="column is-3 has-text-weight-semibold"> <strong> {% translate "Užklausos API adresas" %}: </strong> </div>
+            <div class="column is-7">{{ request_history.endpoint }}</div>
+        </div>
+
+
+        <div class="columns is-mobile mb-1">
+            <div class="column is-3 has-text-weight-semibold"> <strong> {% translate "HTTP metodas" %}: </strong> </div>
+            <div class="column is-7">{{ request_history.method }}</div>
+        </div>
+
+        <div class="columns is-mobile mb-1 is-vcentered">
+            <div class="column is-3 has-text-weight-semibold">
+                <strong> {% translate "Rezultatas" %}: </strong>
+            </div>
+
+            {% if request_history.result not in "UPDATED STATUS_ALIVE UPDATED_WITHOUT_CHANGES" %}
+            <div class="column is-7 is-flex is-align-items-center has-text-danger">
+                    <span class="icon is-small mr-1">
+                        <i class="fas fa-exclamation-circle"></i>
+                    </span>
+                {% translate "NESĖKMINGA" %}
+            </div>
+            {% else %}
+            <div class="column is-7 is-flex is-align-items-center has-text-success">
+                    <span class="icon is-small mr-1">
+                        <i class="fas fa-check-circle"></i>
+                    </span>
+                {% translate "SĖKMINGA" %}
+            </div>
+            {% endif %}
+        </div>
+
+
+        <div class="columns is-mobile mb-1 is-vcentered">
+            <div class="column is-3 has-text-weight-semibold">
+                <strong> {% translate "Statusas (HTTP)" %} </strong>
+            </div>
+
+            {% if request_history.result in "UPDATED STATUS_ALIVE UPDATED_WITHOUT_CHANGES" %}
+            <div class="column is-7 is-flex is-align-items-center has-text-success">
+                <span class="icon is-small mr-1">
+                    <i class="fas fa-check-circle"></i>
+                </span>
+                {{ request_history.get_result_display }}
+            </div>
+            {% else %}
+            <div class="column is-7 is-flex is-align-items-center has-text-danger">
+                <span class="icon is-small mr-1">
+                    <i class="fas fa-exclamation-circle"></i>
+                </span>
+                {{ request_history.get_result_display }}
+            </div>
+            {% endif %}
+        </div>
+
+    </div>
+</div>
+
+<hr style="border: none; border-top: 1px solid #ddd; margin: 2rem 0;">
```

Comment:
Is the extra styling on `<hr>` really needed? Just asking since the ones I added did not :D